chapters/challengeflag.adoc

```diff
@@ -0,0 +1,12 @@
+== Challenge Flag
+
+In American football there is a challenge flag. A team can challenge a decision of the referee:
```

Comment:
I would move the american football part to a note (sth. like: This is inspired by challenge flags in american football).